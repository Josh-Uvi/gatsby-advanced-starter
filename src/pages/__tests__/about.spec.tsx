import * as React from "react";
import "jest-styled-components";
import { render } from "@testing-library/react";
import AboutPage from "../about";

describe("page component AboutPage", () => {
  it("renders correctly", () => {
    expect.assertions(1);
    const { asFragment } = render(<AboutPage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
