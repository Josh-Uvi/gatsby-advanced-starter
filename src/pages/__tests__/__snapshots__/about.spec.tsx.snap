// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`page component AboutPage renders correctly 1`] = `
<DocumentFragment>
  .c0 {
  min-height: 100vh;
  min-width: 100vw;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 20px;
}

@media (min-width:564px) {

}

<div
    class="c0"
  >
    <div
      class="about-container"
    />
  </div>
</DocumentFragment>
`;
